projects/step-frontend/src/lib/plugins-initializer/plugins-initializer.ts

```diff
@@ -3,14 +3,18 @@ import { LegacyPluginDefinition } from './shared/legacy-plugin-definition';
 import { MicrofrontendPluginDefinition } from './shared/microfrontend-plugin-definition';
 import { registerLegacyPlugins } from './register-legacy-plugins';
 import { registerMicrofrontendPlugins } from './register-microfrontend-plugins';
+import { PluginInfoRegistryService } from '@exense/step-core';
 
-// TODO: modify in BE and remove override
-const OVERRIDE_PLUGINS = new Map<string, string>([['multitenancy', 'multitenancy/remoteEntry.js']]);
+const OVERRIDE_PLUGINS = new Map<string, string>();
+
+// Ignore original multitenancy since it's become part of enterprise core
+const IGNORE_PLUGINS: ReadonlyArray<string> = ['multitenancy'];
 
 // For testing purposes only
 // Allows to add plugins, that don't returned from BE
-const ADDITIONAL_PLUGINS: ReadonlyArray<string> = [
-  // Add strings like this 'pluginName/remoteEntry.js'
+const ADDITIONAL_PLUGINS: ReadonlyArray<MicrofrontendPluginDefinition> = [
+  // Add object like this {name: 'pluginName', entryPoint: 'pluginName/remoteEntry.js' }
+  { name: 'stepEnterpriseCore', entryPoint: 'step-enterprise-core/remoteEntry.js' },
```

Comment:
@dvladir  So I guess we want to send this from the BE, right? Also, we need a way to manage which plugins are loaded. It is also ok, to use a flag to check if the BE has these plugins activated. Did you think about this?

Reply:
@neogucky Right now stepEnterprizeCore analyzes the response from '/rest/app/plugins/' to determine, should it include one of it parts or not.
Probably in future, it will be better to use additional endpoint to determine it.

Yes, the link to stepEnterpriseCore, should be also added to that rest endpoint response